### PR TITLE
Add autocomplete to the Version query builder field

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Builds on https://github.com/web-platform-tests/wpt.fyi/pull/689, using the same technique as SHA.

## Review Information
Open the query builder, see autocomplete in the `Version` field for each product.

### CODE REVIEW

File: webapp/components/test-runs-query-builder.html [hunk 1]
```diff
@@ -383,6 +401,52 @@ <h3>
...
+          return;
+        }
+        if (version) {
+          versions = versions.filter(s => s.startsWith(version));
+        }
+        versions = versions.slice(0, 10);
+        // Check actually different from current.
+        const curr = this.versionsAutocomplete || [];
+        if (curr.length === versions.length) {
+          let same = true;
```
Comment (mdittmer) [inline webapp/components/test-runs-query-builder.html]: This looks correct. Would the code be simpler if we added versions to a set and check its length?

File: webapp/components/test-runs-query-builder.html [hunk 2]
```diff
@@ -383,6 +401,52 @@ <h3>
           labels: (product.labels || []).filter(l => !CHANNELS.has(l)).concat(channel)
         });
       }
+
```
Comment (mdittmer) [inline webapp/components/test-runs-query-builder.html]: It's hard to keep the flow straight my method name alone in this block of methods. Could we add short header comments like `// Respond to [foo] changing by [action]`, where `[foo]` and `[action]` are more than just a property or URL name. E.g., "Respond to list of versions to be suggested in data list changing by ..."

File: webapp/components/test-runs-query-builder.html (hunk 1 above)
Comment (lukebjerring) [reply webapp/components/test-runs-query-builder.html]: Simplified.

File: webapp/components/test-runs-query-builder.html (hunk 2 above)
Comment (lukebjerring) [reply webapp/components/test-runs-query-builder.html]: Done.

**Comment by lukebjerring** [comment]
Error is just announcer deployment, quota issue. Force-merging.

**Pull request merged**